src/mpyl/backstage/backstage.py

```diff
@@ -0,0 +1,141 @@
+"""Functions to generate backstage components"""
+
+import logging
+import os
+
+import yaml
+
+from ..plan.discovery import find_projects
+from ..project import load_project, Project, Target, KeyValueProperty
+from ..steps import deploy
+from ..steps.deploy.k8s.chart import ChartBuilder
+
+logger = logging.getLogger("mpyl")
+
+
+def generate_components(directory: str, repository_url: str) -> None:
+    projects: list[Project] = []
+    for project_yaml_file in find_projects():
+        projects.append(load_project(project_yaml_file, validate_project_yaml=True))
+
+    service_components: list[dict] = []
+    maintainers: list[str] = []
+    project_names = __get_project_names(projects)
+    for project in projects:
+        maintainer = ",".join(
```

Comment:
Are we creating a list of maintainers? I don't think this will work that well, it will create a bunch of groups that have similar maintainers. 

I would opt that we for now choose the first of the list and go from there (i.e teams can reach out if its incorrect)

Or else Backstage is going to look like a mess